pkg/transforms/mqtt.go

```diff
@@ -57,7 +57,8 @@ func (sender MQTTSender) MQTTSend(edgexcontext *appcontext.Context, params ...in
 			return false, token.Error()
 		}
 		edgexcontext.LoggingClient.Info("Sent data to MQTT Broker")
-		edgexcontext.LoggingClient.Debug(fmt.Sprintf("Sent data to MQTT Broker: %X", ([]byte)(data)))
+		edgexcontext.LoggingClient.Trace(fmt.Sprintf("Exported data to MQTT Broker: %X. Corrilation Id is '%s'", ([]byte)(data), edgexcontext.CorrelationID))
```

Comment:
Corrilation should be "Correlation"

Reply:
It might be worth discussing whether the broker type should be a logging key. For example, the app name is included as a key. It's possible we should support the same for transforms so that the log message might contain
`transform=mqtt correlation-id=XYZ123`. Again, semi-structured, easier to grep.